tensorflow/core/common_runtime/dml/dml_common.h

```diff
@@ -21,17 +21,18 @@ limitations under the License.
 #include <chrono>
 
 #ifndef _WIN32
+typedef uint64_t UINT64, ULONG_PTR; // TODO: move to dx headers repo
```

Comment:
Do we have a task/bug number for this?

Reply:
Nah I just gotta open a PR in Jesse's repo.